### PR TITLE
Merge editorialized release notes for 18.7 into `develop`

### PR DESCRIPTION
**Pull request by mokagio**
Notice the longer commit history and `RELEASE-NOTES.txt` because of:

- https://github.com/wordpress-mobile/WordPress-iOS/pull/17471
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17474

I decided to wait for the editorialized release notes to make a PR against `develop` and batch all these changes because none has functional impact on the code.

### CODE REVIEW

File: WordPress/Jetpack/Resources/AppStoreStrings.po
Comment (mokagio) [inline WordPress/Jetpack/Resources/AppStoreStrings.po:18]: As @oguzkocer raised when he covered for me releasing version 18.5, the Jetpack subtitle has been longer that 30 characters all this time.

We've discussed it internally and this is the new, shorter version at 23 characters. Hopefully all localizations will keep it below the limit 🤞

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (mokagio) [inline WordPress/Jetpack/Resources/release_notes.txt]: A possibly clearer version for this could be:

> - Users with Administrator or Editor roles can see the comment authors’ email and IP addresses

But I don't think it's worth chaning 🤔

File: WordPress/Jetpack/Resources/AppStoreStrings.po
Comment (AliSoftware) [reply WordPress/Jetpack/Resources/AppStoreStrings.po:18]: image

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (AliSoftware) [reply WordPress/Jetpack/Resources/release_notes.txt]: I'd be ok with cleaning and using your version honestly.
Especially since the meaning is slightly different (and yours being closer to the actual change). The original suggests that if the comment author is an Admin/Editor then their email IP will be shown to everyone. Yours suggests that Admin/Editor will be able to see email and IP of every comment's author, which seems more correct.

File: WordPress/Resources/AppStoreStrings.po
Comment (AliSoftware) [inline WordPress/Resources/AppStoreStrings.po]: Ditto about cleaning with your suggested edit above.

**Review by AliSoftware — APPROVED** [review]
✅  18.6.1 hotfix branch commits: `RELEASE-NOTES.txt` up-to-date but, as expected, no other change from the hotfix because the only change was Gutenberg `v1.65.1` but `18.7` is already on `v1.66.0`, so that checks out.
✅  `release_notes.txt` files updated for WP and JP
✅  `AppStoreStrings.po` files updated for WP and JP

LGTM, but I think it's worth applying the edit that you yourself suggested. It's a cheap change and makes sense to me by avoiding confusion or misinterpreting the impact (even if it's unlikely that we'd get, like, support tickets for people concerned about it or whatnot, better be clear anyway).
For that reason I've disabled auto-merge to let you apply it before merging.

**Review by AliSoftware — DISMISSED** [review]
As discussed with @jkmassel we actually need to add some items to the notes (like the fix about MSWord), I'll push new commits in a couple of minutes to fix that and ping the relevant people, but wanted to "Request changes" ASAP in the meantime to prevent early merge

**Comment by AliSoftware** [comment]
@mokagio I've pushed a new commit 03daea45a8564bff7aaee40617736e170229c3f4 in order to integrate the release note items that were added in https://github.com/wordpress-mobile/WordPress-iOS/pull/17474 post-code-freeze, and were thus not part of the draft which was sent for editorial.

I've borrowed the wording for those from the Editorial'd WPAndroid notes (see https://github.com/wordpress-mobile/WordPress-Android/pull/15584) – as those were included in the draft notes and Editorial'd… before we removed the MSWord item from them.

I've also added periods at the end of each bullet point in the list.

\cc @jkmassel @twstokes @fluiddot

**Comment by AliSoftware** [comment]
Just realized I didn't apply the suggestion about "comments email for Admin roles" item, I'll push another commit with that in a minute.
[EDIT] Done via 22e2a30. This is now ready for review again.

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (AliSoftware) [reply WordPress/Jetpack/Resources/release_notes.txt]: Fixed in 22e2a30

File: WordPress/Resources/AppStoreStrings.po
Comment (AliSoftware) [reply WordPress/Resources/AppStoreStrings.po]: Fixed in 22e2a30

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt]: I considered moving this added line at the very end, it being a bugfix as opposed to the other items being feature improvements. IMHO it would have put less emphasis on the bugfix if I did so, which could have been nice.

But otoh, the wording that was originally provided by Editorial ends with "Finally, we updated the UI […]" so didn't seem right to add an item / line after that "Finally" line…

Any suggestions to find a better balance welcome.

Comment (jkmassel) [reply WordPress/Jetpack/Resources/release_notes.txt:3]:
```suggestion
We fixed an issue that made the interface crash when users copied and pasted lists from Microsoft Word.
```

File: RELEASE-NOTES.txt
Comment (fluiddot) [inline RELEASE-NOTES.txt:10]: This entry is duplicated, as this issue was fixed in `18.6.1`, so we can remove it.

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (jkmassel) [reply WordPress/Jetpack/Resources/release_notes.txt]:
```suggestion
We fixed an issue that made the interface crash when users copied and pasted lists from Microsoft Word.
```

To align with what's in the `.po` file?

File: RELEASE-NOTES.txt
Comment (AliSoftware) [reply RELEASE-NOTES.txt:10]: Yeah I'm not sure how @mokagio usually handles this.

In WPAndroid I've never added a `RELEASE-NOTES.txt` entry for hotfixes like he did for `18.6.1` (because those release notes don't go thru the usual process for hotfixes anyway given their timely nature), so we don't have the issue as we don't have entries for `18.6.1` anyway.

In WPiOS Gio started to do this for hotfixes which has its justifications: I actually like it given that `RELEASE-NOTES.txt` is also for us devs to keep track of when each feature/fix was shipped). But then this raises the question as to how we should cherry-pick the release notes for those cases, as it adds extra churn (both release notes will always be present in that case, once coming from the `release/18.6.1` hotfix branch since we started to add sections for those in WPiOS, and another copy coming from the `develop` branch that the `release/18.7` is cut from, before the hotfix branch gets merged later…).

So that means that when we merge back the `release/18.6.1` hotfix branch into `release/18.7` current release to resync everything, we'd have to add an extra commit to clean up the `RELEASE-NOTES.txt` file as well to remove the duplicates… not ideal either.

---

I'm ok either way (to clean them up and remove those 2 entries from that `REALEASE-NOTES.txt` file here as you suggest, or to keep them to enforce the idea that they were cherry-picked in both hotfix and final?)… but I'll let @mokagio decide on this one given how he handles hotfix release notes in general.

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (AliSoftware) [reply WordPress/Jetpack/Resources/release_notes.txt]: Nice catch! I indeed messed up between various copy/pastes; removed the `also` in b305045.

File: RELEASE-NOTES.txt
Comment (jkmassel) [reply RELEASE-NOTES.txt:10]: I'm ok to mention it with this release so folks know it's been fixed (because the note wouldn't be translated for folks in 18.6.1, so they may not know).

Thanks @fluiddot!!

**Review by jkmassel — APPROVED** [review]
I'm good with this, but I'll let @mokagio merge

File: RELEASE-NOTES.txt
Comment (AliSoftware) [reply RELEASE-NOTES.txt:10]: Oh we're not talking about removing those from the 18.7 release notes in `AppStoreStrings.po` itself, we all agree the entries should still go into the 18.7 release notes that will be translated and displayed to end users on AppStore 🙂

The question was more about should we remove the duplicate entry in the developer-facing `RELEASE-NOTES.txt` (even if we keep it in the `.po` files) 🤷 

That's not critical either way given that's only the internal notes file (which is why I was inclined to let @mokagio do as he prefers)

**Comment by mokagio** [comment]
Screen Shot 2021-11-18 at 2 03 01 pm

😭

**Comment by mokagio** [comment]
Thanks @AliSoftware @jkmassel @fluiddot for the help moving this forward.

Going to solve the merge conflicts, give the diff a last pass, and merge. Unfortunately, this will need to be done in a _new_ PR.

File: RELEASE-NOTES.txt
Comment (mokagio) [reply RELEASE-NOTES.txt:10]: I've decided to remove the "duplicated" entry. More on the rationale here: [`b2ae4fc` (#17483)](https://github.com/wordpress-mobile/WordPress-iOS/pull/17483/commits/b2ae4fcfce7f0d81c82a7dc06f9f2f80ac7474f8)

If this was a library, I would have pushed back, but I think in this case simplicity trumps my own pedantry.

**Comment by AliSoftware** [comment]
Closing this one in favor of https://github.com/wordpress-mobile/WordPress-iOS/pull/17492 which fixes the conflict

**Pull request closed without merging**
